reference/docs-conceptual/setup/Installing-PowerShell-Core-on-Linux.md

```diff
@@ -613,25 +591,12 @@ wget http://security.debian.org/debian-security/pool/updates/main/o/openssl/libs
 sudo dpkg -i libssl1.0.0_1.0.1t-1+deb8u6_amd64.deb
```

Comment:
libicu needs to be updated to libicu60

Reply:
libssl url changed:
http://security.debian.org/debian-security/pool/updates/main/o/openssl/libssl1.0.0_1.0.1t-1+deb8u9_amd64.deb

Maybe add a note that you can find the current url here:
https://packages.debian.org/jessie/amd64/libssl1.0.0/download